test/support/pet.ex

```diff
@@ -43,6 +50,17 @@ defmodule Flop.Pet do
     belongs_to :owner, Owner
   end
 
+  def test_custom(query, %Flop.Filter{value: value} = filter, opts) do
+    :options = Keyword.fetch!(opts, :some)
+    send(self(), {:filter, {filter, opts}})
+
+    if value == "some_value" do
+      where(query, false)
+    else
+      query
+    end
+  end
```

Comment:
Can we come up with some actual custom filter we can apply? Maybe `reverse_name` or something like that? Then we can add that functionality to the `filter_pets` function as well and use a property test as in the rest of the test module.